purchase_variant_configurator/views/inherited_purchase_order_views.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record model="ir.ui.view" id="purchase_order_extended_form_view">
+        <field name="name">purchase.order.extended.form</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="priority" eval="20" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='order_line']/tree" position="attributes">
+                <!-- force using the form -->
+                <attribute name="editable" />
+            </xpath>
+            <xpath expr="//field[@name='order_line']/form/group[1]" position="before">
+                <group>
+                    <field name="state" invisible="1" />
+                    <!-- HACK: readonly condition is set with an `in` clause instead of a `not in` for allowing on new created lines to be writeable, as
+                             state field is a related one that is not filled until saved -->
+                    <field
+                        name="product_tmpl_id"
+                        attrs="{'invisible': [('display_type', '!=', False)], 'readonly': [('state', 'in', ('purchase', 'to approve','done', 'cancel'))]}"
+                        domain="[('purchase_ok','=',True)]"
+                    />
+                    <field
+                        name="product_attribute_ids"
+                        widget="one2many"
+                        context="{'default_owner_model': 'purchase.order.line'}"
+                        attrs="{'invisible': [('product_attribute_ids','=',[])], 'readonly': [('state', 'in', ('purchase', 'to approve','done', 'cancel'))]}"
+                        nolabel="1"
+                        colspan="2"
+                    >
+                        <tree create="0" delete="0" editable="1">
+                            <field name="owner_model" invisible="1" />
```

Comment:
```suggestion
                            <field name="owner_model" column_invisible="True" />
```
Similar in other fields to avoid displaying them.
![ejemplo](https://github.com/user-attachments/assets/d46bce3a-6881-4816-b7de-6c28fcc0c184)